### ISSUE TITLE
A wrapper label with visibility:hidden breaks upload testing with Selenium

### ISSUE THREAD

**Issue by n1313** [danialfarid/ng-file-upload#1423]
TL;DR: please add a unique identifier to the `<label>` wrapper so I could easily locate it in my upload tests. 

Long story: I have a selenium test for my upload form. Long ago I had some troubles with it because the `input[type=file]` generated by ng-file-upload is created hidden, but since version 5 (I think) when proper data-ids were introduced it became less of an issue because I could easily locate it and change the visibility settings manually before proceeding with the test. Now after updating to version 12 the problem is back again (in Firefox), but this time it is because of the hidden `<label>` that wraps around the input (#1176). I fixed my tests by manually locating the input, then locating its immediate parent, and changing its visibility in the same manner as I do with the input. But that is not very elegant so I'd like to have an easier way of locating that element. A unique id derived from the id on the input would be very useful here.

**Comment by danialfarid** [comment]
Since 12.1.0 you can find it by `ngf-label-<element id>`.
